### DISCUSSION TITLE
Using `mutate()` to clear cache doesn't work if called from a external package

### DISCUSSION THREAD

**Discussion by vegidio** [vercel/swr#2485] [Q&A]
I've been using `mutate` to clear the cache in my React Native application:

```typescript
import { mutate } from 'swr';
function clearCache() {
    mutate(() => true, undefined, { revalidate: false });
}
```

and this works as expected when I call this function in my application.

However, I recently created a private NPM package that contains many helper & utility functions to ease the development of apps for my organization, and I moved the function above to this private package:

```typescript
import { clearCache } from 'my-private-package';
clearCache();
```

But now, if I import this private package in another project and try to use the `clearCache()` function, the SWR cache is **not** cleared.

Does anyone have any idea why `clearCache()` function above works when it's implemented directly in my project, but it no longer works when it's implemented and called from a different package?

**Comment by koba04** [comment]
> However, I recently created a private NPM package that contains many helper & utility functions to ease the development of apps for my organization, and I moved the function above to this private package:

This might be caused by having multiple `swr` package in your application.
